vsample type mismatch

from

https://www.stats.ox.ac.uk/pub/bdr/LTO/randomizr.out

```
* installing *source* package ‘randomizr’ ...
** package ‘randomizr’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
make[2]: Entering directory '/data/gannet/ripley/R/packages/tests-LTO/randomizr/src'
gcc -I"/data/gannet/ripley/R/LTO9/include" -DNDEBUG   -I/usr/local/include   -fpic  -g -O2 -Wall -pedantic -mtune=native -flto -c onload.c -o onload.o
gcc -I"/data/gannet/ripley/R/LTO9/include" -DNDEBUG   -I/usr/local/include   -fpic  -g -O2 -Wall -pedantic -mtune=native -flto -c restrictedparts.c -o restrictedparts.o
gcc -I"/data/gannet/ripley/R/LTO9/include" -DNDEBUG   -I/usr/local/include   -fpic  -g -O2 -Wall -pedantic -mtune=native -flto -c vsample.c -o vsample.o
gcc -shared -g -O2 -Wall -pedantic -mtune=native -flto -fpic -L/usr/local/lib64 -o randomizr.so onload.o restrictedparts.o vsample.o
onload.c:9:13: warning: type of ‘randomizr_vsample’ does not match original declaration [-Wlto-type-mismatch]
    9 | extern SEXP randomizr_vsample(SEXP, SEXP);
      |             ^
vsample.c:5:6: note: type mismatch in parameter 2
    5 | SEXP randomizr_vsample(SEXP pmat) {
      |      ^
vsample.c:5:6: note: type ‘void’ should match type ‘struct SEXPREC *’
vsample.c:5:6: note: ‘randomizr_vsample’ was previously declared here
make[2]: Leaving directory '/data/gannet/ripley/R/packages/tests-LTO/randomizr/src'
installing to /data/gannet/ripley/R/packages/tests-LTO/Libs/randomizr-lib/00LOCK-randomizr/00new/randomizr/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (randomizr)
Time 0:13.99, 4.95 + 0.53
```

--- a/src/onload.c
+++ b/src/onload.c
@@ -6,7 +6,7 @@
 
 /* .Call calls */
 extern SEXP randomizr_restrictedparts(SEXP, SEXP);
-extern SEXP randomizr_vsample(SEXP, SEXP);
+extern SEXP randomizr_vsample(SEXP);
 
 
 

